engineio/base/packet_test.go

```diff
@@ -1,44 +1,46 @@
 package base
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
+var testPacketTypeCases = []struct {
```

Comment:
As other tests don't use these cases, I prefer to put it in the `testPacketTypeCases` function.